pkg/gofr/http/responder.go

```diff
@@ -23,6 +24,12 @@ type Responder struct {
 // status codes and formatting responses as JSON or raw data as needed.
 func (r Responder) Respond(data interface{}, err error) {
 	statusCode, errorObj := r.HTTPStatusFromError(err)
+	if errorObj != nil && len(errorObj.(map[string]interface{})) != 0 {
```

Comment:
This is not required as we have the custom errors package in pipeline #604. Also we don't want to check strings to decide status codes.

Reply:
Hi @vipul-rawat, I don't notice it. Thanks for there reference. I'll remove this.